global/prometheus-alertmanager/templates/config.yaml

```diff
@@ -49,7 +49,7 @@ data:
         match_re:
           service: k8s|etcd|prometheus|dns|node
           cluster: {{ . }}
-          severity: critical
+          severity: warning,critical
```

Comment:
Remove please. Too much spam in region channels with warnings being posted there as well.